docs/integrations/builtin/credentials/facebookgraph.md

```diff
@@ -32,7 +26,54 @@ Refer to [Meta's Graph API documentation](https://developers.facebook.com/docs/g
 
 ## Using app access token
 
-To configure this credential, you'll need:
+o configure this credential, you'll need a [Meta for Developers](https://developers.facebook.com/){:target=_blank .external-link} account and:
+
+- An app **Access Token**
+
+There are two steps in setting up your credential:
+
+1. [Create a Meta app](#create-a-meta-app) with the products you need to access.
+2. [Generate an App Access Token](#generate-an-app-access-token) for that app.
+
+Refer to the detailed instructions below for each step.
+
+### Create a Meta app
+
+To create a Meta app:
+
+1. Go to the Meta Developer [App Dashboard](https://developers.facebook.com/apps){:target=_blank .external-link} and select **Create App**.
+2. If you have a business portfolio and you're ready to connect the app to it, select the business portfolio. If you don't have a business portfolio or you're not ready to connect the app to the portfolio, select **I don’t want to connect a business portfolio yet** and select **Next**. The **Use cases** page opens.
+3. Select the **Use case** that aligns with how you wish to use the Facebook Graph API. For example, for products in Meta's **Business** suite (like Messenger, Instagram, WhatsApp, Marketing API, App Events, Audience Network, Commerce API, Fundraisers, Jobs, Threat Exchange, and Webhooks), select **Other**, then select **Next**.
+4. Select **Business** and **Next**.
+5. Complete the essential information:
+    * Add an **App name**.
+    * Add an **App contact email**.
+    * Here again you can connect to a business portfolio or skip it.
+1. Select **Create app**.
+1. The **Add products to your app** page opens.
+1. Select **App settings > Basic** from the left menu.
+1. Enter a **Privacy Policy URL**. (Required to take the app "Live.")
```

Comment:
This is one place where I'd typically consider linking to a doc from Meta that confirms this information and explains the requirements more in-depth. I'm unsure about if / how we do this at n8n though. Curious to hear your thoughts.

Reply:
@imchairmanm yes, that would be a good practice, if Meta had docs that were particularly clear. I've certainly done that other places. :) 

My dilemma was this: their docs officially say that ["Business" apps don't need to go into "Live" mode](https://developers.facebook.com/docs/development/release#app-types):
>Business apps do not have app modes and rely exclusively on access levels.

But in my testing, our credentials won't connect if the app isn't in Live mode. So I was hesitant to link to official docs that might mildly contradict what we're instructing folks to do. 

More info on the modes is available [here](https://developers.facebook.com/docs/development/build-and-test/app-modes).

My inclination would be to add both of these links in the "Refer to xxx" section as additional resources rather than interrupting the stream of instructions here, given my own misgivings about their docs. Wdyt?

This one's your call at this point.